dispatch/static/manager/src/js/reducers/AuthReducer.js

```diff
@@ -38,4 +39,12 @@ reducer.handle(fulfilled(types.AUTH.DELETE_TOKEN), () => {
   return initialState
 })
 
+reducer.handle(fulfilled(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: true })
+})
+
+reducer.handle(rejected(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: false })
```

Comment:
does rejected include timeouts/other connection failures or only 4xx errors returned by the server? I don't like the idea of kicking people out of the app because their internet is inconsistent.